server/services/crmDisplayService.ts

```diff
@@ -107,7 +107,7 @@ export default class CrmDisplayService {
   }
 
   private getApiFieldValue<T extends CrmResponse>(crmResponse: T, apiFieldName: string): string {
-    return _.get(crmResponse, apiFieldName) || ''
+    return _.get(crmResponse, apiFieldName) || _.get(crmResponse, `formDetails.${apiFieldName}`) || ''
```

Comment:
When BE structure has been changed for CRM 5 API as well, then this code will be changed to as follows:-
`return _.get(crmResponse, `formDetails.${apiFieldName}`) || ''`